mysql-test/suite/galera/t/galera_retry_applying2.test

```diff
@@ -0,0 +1,40 @@
+#
+# Test retrying applying of a transaction
+#
+
+--source include/galera_cluster.inc
+--source include/have_innodb.inc
+
+#
+# Slave retries applying of a transaction multiple times. All retry
+# attempts fail, and the applying will fail with the expected error.
+#
+
+CREATE TABLE t2 (f1 INTEGER PRIMARY KEY DEFAULT 0, f2 char(12));
+
+--connection node_2
+CALL mtr.add_suppression("WSREP: Event .* Write_rows_v1 apply failed:");
+CALL mtr.add_suppression("WSREP: Inconsistency detected");
+CALL mtr.add_suppression("WSREP: Failed to apply write set");
+
+SET GLOBAL wsrep_applier_retry_count = 2;
+SET GLOBAL debug_dbug = "+d,innodb_insert_fail:o,/dev/null";
+
+--connection node_1
+SLEEP 1; 
```

Comment:
there should be no reason for sleep here, please remove it.
It is possible to add wait_condition for connection node_2 to wait until table t2 has been replicated there, but as we have wsrep_sync_wait=15 by default, there should be no absolute reason to wait for the replication

Reply:
Fixed as suggested.